src/styles/getMuiTheme.js

```diff
@@ -224,4 +245,11 @@ export default function getMuiTheme(baseTheme, muiTheme) {
       borderColor: palette.borderColor,
     },
   }, muiTheme);
+
+  const transformers = computedTheme.transformers ? computedTheme.transformers :
```

Comment:
I feel like it would be more readable to get the transformers as function argument. 

like:

```js
getMuiTheme(baseTheme, muiTheme, transformers)
```
Since people are very unlikely to want to modify transformers.